source/includes/_cvt_events.md

```diff
@@ -7,29 +7,90 @@
 |---|---|
 | GET | /client_reports/cvt_events/[gmaid]?[query_params]|
 
-### API Name: CVT Events
+### API Name: CVT/Web Events
+
+### Summary
+
+This is an API that can be used by customers for whom we are pages of importance to you (CVT/Web Events). All metrics are reported against the CVT/Web Event.
```

Comment:
Grammar not quite right here?  Perhaps "customers who have indicated that certain pages are of specific importance".  Ask someone in marketing.

Reply:
I spoke with my friend in Marketing who gave some insight into how/why CVTs are set up which can be summarized as:

"This is an API that can be used by customers who have indicated that certain pages or actions of specific importance will benefit the performance of campaigns through more informed and meaningful conversions."